scripts/xmpro-website-scripts/scrape-xmpro-website-usecases.py

```diff
@@ -0,0 +1,147 @@
+import os
+import requests
+from bs4 import BeautifulSoup
+from pathvalidate import sanitize_filename
+import json
+
+# Function to save content to Markdown file
+def save_to_md(title, content, url, folder_path):
+    try:
+        # Sanitize the title to ensure it's suitable for use as a filename
+        sanitized_title = sanitize_filename(title.strip().replace("&", ""))
+        
+        # Truncate the filename to a maximum of 20 characters
+        truncated_title = sanitized_title[:20]
+        
+        # Convert title to lowercase and replace spaces with "-"
+        filename = os.path.join(folder_path, f"{truncated_title.lower().replace(' ', '-')}.md")
+
+        with open(filename, 'w', encoding='utf-8') as file:
+            file.write(f"# {title}\n\n")
+            file.write(f"URL: {url}\n\n")
+            file.write(content)
+        print(f"Content saved to {filename}")
+        return {'title': title, 'filename': filename}
+    except Exception as e:
+        print(f"Error occurred while saving to file: {e}")
+
+# Function to scrape content from each page
+def scrape_page(url, folder_path):
+    try:
+        # Send a GET request to the page URL
+        response = requests.get(url)
+
+        # Check if the request was successful
+        if response.status_code == 200:
+            # Parse the HTML content
+            soup = BeautifulSoup(response.text, "html.parser")
+            
+            # Find the title of the page
+            title_elem = soup.find("title")
+            if title_elem:
+                title = title_elem.text.strip()
+
+                # Find the portfolio-inner div
+                portfolio_inner_div = soup.find("div", class_="portfolio-inner")
+
+                if portfolio_inner_div:
+                    # Find all content elements within the portfolio-inner div
+                    content_elements = portfolio_inner_div.find_all(["p", "h1", "h2", "h3", "h4", "img"])
+
+                    # Initialize markdown content
+                    markdown_content = ""
+
+                    # Append content to the markdown content preserving order
+                    for element in content_elements:
+                        if element.name == "img":
+                            prop = "src"
+                            if "data-src" in element.attrs:
+                                prop = "data-src"
+                            w = element.get("width")
```

Comment:
@jacksonmcguirkxmpro 

Lets consider the case if width or any other attribute doesn't exist

Reply:
If the width or other attributes do not exit, rather then stopping the code or not showing the image, it continues to show the images but at its original width.